articles/quickstart/native/ios-swift/03-user-sessions.md

```diff
@@ -161,25 +160,26 @@ Lock
 
 ### Use the refreshToken to get a new idToken
 
-We can use the `func renew(withRefreshToken refreshToken: String)` method in `Auth0` to yield fresh user's credentials.
+You can use the `delegation` method in `Auth0` to yield fresh user's credentials using the refreshToken.
```

Comment:
the wording for "fresh user's credentials" is a bit weird here. would it be better to say  "You can use the `refreshToken` in the `delegation` method to renew the user's `idToken`.